dds/DCPS/transport/framework/TransportRegistry.cpp

```diff
@@ -661,10 +661,8 @@ TransportRegistry::create_new_transport_instance_for_participant(DDS::DomainId_t
   OPENDDS_STRING inst_name = cfg->instances_[0]->name() + "_" + instance_config_name;
   instance_config_name = "transport_config_" + instance_config_name;
 
-  OpenDDS::DCPS::TransportConfig_rch config =
-    TheTransportRegistry->create_config(instance_config_name);
-  OpenDDS::DCPS::TransportInst_rch inst =
-    TheTransportRegistry->create_inst(inst_name, "rtps_udp");
+  OpenDDS::DCPS::TransportConfig_rch config = create_config(instance_config_name);
```

Comment:
Can these fail? Do we need to check if these RCH are valid and return false if they are, or something similar?